docs/designers-developers/developers/block-api/block-context.md

```diff
@@ -0,0 +1,70 @@
+# Block Context
```

Comment:
I wrote this toward the end of my work day, so I'll probably want to do another proof-read pass when my mind is in a fresher state 😅

Reply:
Some general grammar improvements in 8ef999c92.